src/components/Inventory/InventoryCreate.tsx

```diff
@@ -142,6 +176,7 @@ export default function CreateInventory(): JSX.Element {
                   value={record.addedDate}
                   onChange={(value) => changeDate('dateAdded', value)}
```

Comment:
this should be `addedDate` .. oh noes :(
that's the bug

Reply:
Oh, yes, for inventory create this is a bug. Will fix it! For editing a batch, the value is not saved because the api doesn't accept that value